addons/web_editor/tests/test_converter.py

```diff
@@ -151,6 +154,10 @@ def test_float(self):
         self.field_roundtrip('float', 42.567890)
         self.field_roundtrip('float', 324542.567890)
 
+    def test_float_time(self):
+        self.field_roundtrip('float_time', '42:00', 42.00, True)
+        self.field_roundtrip('float_time', '42:80', 43.33, True)
```

Comment:
No, it should not raise an error because `42:80` is the value for the `float_time` field. This is an unusual time format because only 60 minutes in an hour exist. However, the system converts 80 minutes into 1.33 hours and adds it to the 42 hours, resulting in 43.33.

Reply:
How we can get such a value in real life? User types it somewhere?

Reply:
User can add this type of value in fields like `completion_time` in `slide.slide` model and `appointment_duration` in `appointment.type` model etc. 

see-
https://drive.google.com/file/d/1Gjb7ZsLYolUdbm9cT2wA9TMc1Qg7LNr5/view?usp=drive_link

also see the code-
https://github.com/odoo/odoo/blob/2eeab8fc71600ebd61767408acd8eb8ff1b63a0c/odoo/tools/misc.py#L1536-L1547

Reply:
I don't think it's a good idea to let user to input values like 42:75. They may confuse colon and point. 
For example, the want to input 42 hours, 45 minutes. It's 42.75 hours in float. However, if user types it by mistake with a colon, it will have completely different  value. We should raise an error and don't let user to make such a mistake